app/assets/javascripts/client/resources/InventoryInvitable.js

```diff
@@ -0,0 +1,8 @@
+PDRClient.factory('InventoryInvitable', ['$resource', 'UrlService', function($resource, UrlService) {
```

Comment:
This resource isn't using the IIFE style, and it's relying on the `PDRClient` variable.